src/module/helpers-dice.js

```diff
@@ -208,7 +208,7 @@ const OseDice = {
     result.target = data.roll.thac0;
     const targetActorData = data.roll.target?.actor?.system || null;
 
-    const targetAc = data.roll.target ? targetActorData.ac.value : 20;
+    const targetAc = data.roll.target ? targetActorData.ac.value : 9;
     const targetAac = data.roll.target ? targetActorData.aac.value : -20;
```

Comment:
If you're changing AC, AAC would have to change as well (they always add up to 19)

Reply:
If the idea is to always roll damage when there is no target selected (as opposed to conforming to the THAC0 Table) we should instead just unclamp the THAC0 value.  I think the confusion only comes from the fact it shows as hitting AC 9 regardless of the dice number that was rolled.

https://github.com/vttred/ose/blob/339b156cefd6cc17262a89f95b9d43b2024dd01c/src/module/helpers-dice.js#L235-L241

Reply:
https://github.com/vttred/ose/pull/504/commits/f3b7975fde5eb91906879397d0ec9cdceb66016c 
- Unclamped the thac0 result so it will show values outside the original thac0 table.
- Updated success check so if the target is null it will always display attack results if a target is not selected. (Natural 1s still produced failures before.)

https://github.com/vttred/ose/pull/504/commits/f17b7b2afdf7c4923beb856d6720411a05a8a37e
- Changed default target ac and aac values, but these are now irrelevant as they are unused.